### PR TITLE
[@types/mongodb] Fix Cursor<T> type (close() method)

### PR DESCRIPTION
**Pull request by kayk1m**
`options` parameter of the close() method is an 'optional parameter' but it was mis-typed.

- document url: https://mongodb.github.io/node-mongodb-native/3.6/api/Cursor.html#close
- occurred error:
image

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://mongodb.github.io/node-mongodb-native/3.6/api/Cursor.html#close

### CODE REVIEW

**Review by LinusU — APPROVED** [review]
👍

**Comment by Jeonghyun-Kim** [comment]
close() of AggregationCursor<T> needs to return `Promise<void>`, not `Promise<AggregationCursorResult`>

https://mongodb.github.io/node-mongodb-native/3.6/api/AggregationCursor.html#close

File: types/mongodb/index.d.ts [hunk 1]
```diff
@@ -4308,7 +4309,7 @@ export class AggregationCursor<T = Default> extends Cursor<T> {
      * @returns Promise if no callback is passed
      * @see https://mongodb.github.io/node-mongodb-native/3.6/api/AggregationCursor.html#close
      */
-    close(): Promise<AggregationCursorResult>;
+    close(): Promise<void>;
```
Comment (LinusU) [inline types/mongodb/index.d.ts:4312]: Hmm, why was this changed to `void`? 🤔

Comment (Jeonghyun-Kim) [reply types/mongodb/index.d.ts:4312]: Oops, my bad. reverted it

**Review by LinusU — APPROVED** [review]
Nice 👍
